os-server/src/main/java/app/openschool/common/security/SecurityConfiguration.java

```diff
@@ -18,7 +17,6 @@
 
 @Configuration
 @EnableWebSecurity
-@Profile("!openapi")
```

Comment:
when we run with test profile 2 security configs will be in the context, one will override another

Reply:
But now we have only one security config class, I've removed the second one. Or there is another reason?